src/XIOPAK.H

```diff
@@ -114,6 +114,11 @@ uintmax_t FC dskDrvFreeSpace();
 SI     FC xffilcmp( const char * fileName1, const char * fileName2 );
 SEC	xfclear(XFILE* xf);
 
+void getPathRootName(const char* path, char* rootName);
+void getPathRootDirectory(const char* path, char* rootDirectory);
+void getPathRootStem(const char* path, char* rootStem);
+void getPathExtension(const char* path, char* fileExtension);
+bool hasExtension(const char* filePath);
```

Comment:
This is a notable deviation from the existing function naming convention in this file. Is it intentional?

Reply:
The pattern is generally "xf", but I'm also referring to the casing. The existing names are all lower case. I'm not saying it is a good style, but if we're going to change style that should be a separate and coordinated effort. I would recommend:

- `xfpathroot`
- `xfpathdir`
- `xfpathbasefilename` or `xfpathstem`
- `xfpathext`
- `xfhasext`